doc-surrealdb_versioned_docs/version-latest/cli/start.mdx

```diff
@@ -160,6 +175,39 @@ SurrealDB supports the ability to startup in strict mode. When running in strict
 surreal start --strict --log debug memory
 ```
 
+## SurrealKV (Beta)
+
+<Since v="v2.0.0" />
+
+SurrealKV is an embedded ACID-compliant key-value storage engine with built-in versioning, that allows for historical or temporal querying. Built entirely in Rust - like the rest of SurrealDB.
+
+Since SurrealKV supports historical/temporal querying, you can user the `VERSION` clause when [SELECTING](/docs/surrealql/statements/select#the-version-clause) or [CREATING](/docs/surrealql/statements/create#version) data to access historical data. Learn more in the [SurrealQL documentation](/docs/surrealql/statements/select#version). 
```

Comment:
```suggestion
Since SurrealKV supports historical/temporal querying, you can user the `VERSION` clause when [selecting](/docs/surrealql/statements/select#the-version-clause) or [creating](/docs/surrealql/statements/create#version) data to access historical data. Learn more in the [SurrealQL documentation](/docs/surrealql/statements/select#version). 
```